From ad66c90ded170fda2d40883303460c857ba1cf21 Mon Sep 17 00:00:00 2001
From: Edward Hervey <edward@centricular.com>
Date: Fri, 11 May 2018 14:19:42 +0200
Subject: [PATCH 10/72] avmux: Remove unneeded constant variable

Since the removal of the gif hacks, the variable was always set to
its default value of FALSE. Just remove it.

CID #1433656
---
 ext/libav/gstavmux.c | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/ext/libav/gstavmux.c b/ext/libav/gstavmux.c
index 907424b..eb93151 100644
--- a/ext/libav/gstavmux.c
+++ b/ext/libav/gstavmux.c
@@ -726,7 +726,6 @@ gst_ffmpegmux_collected (GstCollectPads * pads, gpointer user_data)
   if (best_pad != NULL) {
     GstBuffer *buf;
     AVPacket pkt;
-    gboolean need_free = FALSE;
     GstMapInfo map;
 
     /* push out current buffer */
@@ -757,11 +756,7 @@ gst_ffmpegmux_collected (GstCollectPads * pads, gpointer user_data)
     else
       pkt.duration = 0;
     av_write_frame (ffmpegmux->context, &pkt);
-    if (need_free) {
-      g_free (pkt.data);
-    } else {
-      gst_buffer_unmap (buf, &map);
-    }
+    gst_buffer_unmap (buf, &map);
     gst_buffer_unref (buf);
   } else {
     /* close down */
-- 
2.17.0

